content/applications/general/email_communication/email_servers.rst

```diff
@@ -263,11 +263,45 @@ To edit catchall and bounce aliases, first activate the :ref:`developer mode <de
 Then, go to :menuselection:`Settings --> Technical --> Parameters --> System Parameters` to
 customize the aliases (`mail.catchall.alias` & `mail.bounce.alias`). These types of changes should
 be completed prior to the database going live. If a customer replies after a change is made then the
-system will not recognize the old alias and the reply won't be received.
+system will not recognize the old alias, and the reply will not be received.
 
 By default, inbound messages are fetched every 5 minutes for on-premise databases.
 
 .. note::
    This value can be changed in :ref:`developer mode <developer-mode>`. Go to
    :menuselection:`Settings --> Technical --> Automation --> Scheduled Actions` and look for
    :guilabel:`Mail: Fetchmail Service`.
+
+Allow alias domain system parameter
+-----------------------------------
+
+Incoming aliases are set in the Odoo database to create records by receiving incoming emails. To
+view aliases set in the Odoo database, first activate the :ref:`developer mode <developer-mode>`.
+Then, go to :menuselection:`Settings app --> Technical --> Email section --> Aliases`.
+
+The following system parameter, `mail.catchall.domain.allowed`, set with allowed alias domain
+values, separated by commas, filters out correctly addressed emails to aliases. Setting the
+domain(s) for which the alias can create a ticket, lead, opportunity, etc., eliminates false
+positives where email addresses with only the prefix alias (not the domain) are present.
+
+In some instances, matches have been made in the Odoo database when an email is received with the
+same alias prefix and a different domain on the incoming email address. This is true in the sender,
+recipient, and :abbr:`CC (Carbon Copy)` email addresses of an incoming email.
+
+.. example::
+   When Odoo receives emails that have the name `commercial` prefix alias in the sender, recipient,
+   or :abbr:`CC (Carbon Copy)` email address(es) (e.g. commercial@gmail.com, commercial@odoo.net),
+   the database falsely treats the email as the full `commercial` alias (with a different domain),
+   and therefore, creates a ticket/lead/opportunity/etc..
```

Comment:
Lets add literal formatting to these email addresses to be consistent with the rest of this documentation :)
```suggestion
   or :abbr:`CC (Carbon Copy)` email address(es) (e.g. `commercial\@gmail.com`,
   `commercial\@odoo.net`), the database falsely treats the email as the full `commercial` alias
   (with a different domain), and therefore, creates a ticket/lead/opportunity/etc..
```

Reply:
Agree with Sam's comment.

Also on line 295: either add another period to form a complete ellipses OR remove the second period to end the sentence

```suggestion
   (with a different domain), and therefore, creates a ticket/lead/opportunity/etc...
```